core-tests/src/test/java/org/mule/routing/filters/ExpressionFilterTestCase.java

```diff
@@ -100,20 +96,19 @@ public void testHeaderFilterWithNotNullEL() throws Exception
         assertTrue(filter.accept(message));
     }
 
-    @Ignore("MULE-9341")
```

Comment:
Because as part of this work, i added support for Event with expression filter as it was needed for something else.